tools/run_pr_builder.sh

```diff
@@ -17,18 +17,6 @@ node --version
 # go list is expensive, only do it once.
 GO_LIST="$(go list ./... | grep -v /vendor/ | grep -v /samples/js/)"
 go build ${GO_LIST}
-
-# go test plus build coverage data for upload codecov.io
-rm -rf coverage.txt
-touch coverage.txt
-for d in ${GO_LIST}; do
-    go test -coverprofile=profile.out -covermode=atomic $d
-    if [ -f profile.out ]; then
-        cat profile.out >> coverage.txt
-        rm profile.out
-    fi
-done
+go test ${GO_LIST}
 
 python -m unittest discover -p "*_test.py" -s $GOPATH/src/github.com/attic-labs/noms/tools
-
-bash <(curl -s https://codecov.io/bash) -t ${COVERALLS_TOKEN}
```

Comment:
not sure if you noticed, but you can remove the COVERALLS_TOKEN thing from jenkins after this